### PR TITLE
aligning codec types with webrtc-pc

### PR DESCRIPTION
**Pull request by vr000m**
Fixing issues #77

### CODE REVIEW

**Comment by vr000m** [comment]
One unrelated change was pushed for a reference.

File: webrtc-stats.html [hunk 1]
```diff
@@ -632,12 +632,12 @@
...
               </dd>
               <dt>
-                <dfn><code>parameters</code></dfn> of type <span class=
+                <dfn><code>sdpFmtpLine</code></dfn> of type <span class=
                 "idlMemberType"><a>DOMString</a></span>
               </dt>
               <dd>
                 <p>
-                  From the SDP description line.
+                  The a=fmtp line in the SDP corresponding to the codec, as defined by [[!JSEP]] (section 5.7.).
```
Comment (alvestrand) [inline webrtc-stats.html]: Need to say where in the a=fmtp line this starts (after the colon following the PT).

File: webrtc-stats.js [hunk 2]
```diff
@@ -94,17 +94,13 @@ var respecConfig = {
...
-            href:     "https://tools.ietf.org/html/draft-ietf-xrblock-independent-burst-gap-discard",
-            authors:  [
-                "Varun Singh",
-                "Colin Perkins",
-                "Alan Clark",
-                "Rachel Huang"
-            ],
-            status:   "Internet Draft",
-            publisher:  "IETF"
+        "JSEP": {
```
Comment (alvestrand) [inline webrtc-stats.js]: Check how webrtc-pc references JSEP, and do the same thing.

**Comment by alvestrand** [comment]
Varun to address comments and then merge.
Note: Renames will cause more trouble once Chrome has released, so this may be the last easy rename.

File: webrtc-stats.js (hunk 2 above)
Comment (vr000m) [inline webrtc-stats.js]: the sentence and reference here is verbatim from webrtc-pc

File: webrtc-stats.html (hunk 1 above)
Comment (taylor-b) [inline webrtc-stats.html]: Is that something we'd already decided? It's one of the virtual interim topics.

Comment (vr000m) [inline webrtc-stats.html]: Yes, agreed almost a year ago in Sapporo.

**Pull request merged**